documentation/customisation.md

```diff
@@ -0,0 +1,139 @@
+##########################################################################
+ADD --chown=10001:0 files/ojdbc6.jar /opt/xebialabs/xl-deploy-server/lib/
+```
+
+Please note that there are seperate hotfix directories where you can place hotfix JARs, depending on the type of hotfix. Please consult our support team in case you are uncertain. 
+
+For an overview of how `ADD` and `COPY` works, please refer to [the documentation](https://docs.docker.com/engine/reference/builder/#add)
+
+Once you are satisfied with your Dockerfile, run the following command in the same directory
+
+`docker build -t xl-deploy-custom:9.5.0 .`
+
+Always use [semver](https://semver.org/) to version your docker images. This is to ensure future compatability with one of our other tools, `xl up`.
+
+This command will build and tag a docker image for you. If you would like to run this image locally, you can do this now by running
+
+`docker run -it --rm -p 4601:4601 -e ACCEPT_EULA=Y xl-deploy-custom:9.5.0`
+
+If you would like to host this elsewhere, you have two options
+
+1. (Recommended) [Push this image to a docker registry](https://docs.docker.com/engine/reference/commandline/push/) of your choice. You can either [set up your own registry](https://docs.docker.com/registry/), or use an offering from DockerHub, AWS, GCP and many others. The simplest way of achieving this is to simply run
+  - `docker tag xl-deploy-custom:9.5.0 yourdockerhuborg/xl-deploy-custom:9.5.0`
+  - `docker push yourdockerhuborg/xl-deploy-custom:9.5.0`
+  - (On the node you would like to run the container) `docker run -it --rm -p 4601:4601 -e ACCEPT_EULA=Y yourdockerhuborg/xl-deploy-custom:9.5.0`
+2. By using [`docker export`](https://docs.docker.com/engine/reference/commandline/export/) and [`docker load`](https://docs.docker.com/engine/reference/commandline/load/). This is not a recommended approach as it requires you to move a tar file between different machines. 
+
+
+## XL Release
+
+As with XL Deploy, always remember that the owning user:group combination _must_ be `10001:0` for any files you add.
+
+To start off with, create a `Dockerfile` that resembles the following. 
+
+Please note that certain JARS will go into certain paths. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team. 
+
+NOTE: `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
+
+```docker
+FROM xebialabs/xl-release:9.5.0
+
+###################################################################################
+# PLUGINS                                                                         #
+# Plugins should be placed under ${APP_HOME}/default-plugins/ #
+###################################################################################
+
+COPY --chown=10001:0 files/xld-liquibase-plugin-5.0.1.xldp /opt/xebialabs/xl-release-server/default-plugins/xlr-official/
```

Comment:
- This is XLDeploy plugin .. we can use one of official plugins like below,
xlr-delphix-plugin-9.0.0.jar

Reply:
indeed add an xlr plugins as an example